mmdet/models/anchor_heads/ssd_head.py

```diff
@@ -79,18 +80,25 @@ def __init__(self,
         for k in range(len(anchor_strides)):
             base_size = min_sizes[k]
             stride = anchor_strides[k]
-            ctr = ((stride) / 2., (stride) / 2.)
+            center = ((stride) / 2., (stride) / 2.)
             scales = [1., np.sqrt(max_sizes[k] / min_sizes[k])]
             ratios = [1.]
             for r in anchor_ratios[k]:
                 ratios += [1 / r, r]  # 4 or 6 ratio
-            anchor_generator = AnchorGenerator(
-                base_size, scales, ratios, scale_major=False, ctr=ctr)
+            anchor_generator_args = dict(
+                base_size=base_size,
+                scales=scales,
+                ratios=ratios,
+                scale_major=False,
+                center=center)
+            base_anchor_generator = build_anchor_generator(
```

Comment:
Seems no reason to rename it to `base_anchor_generator`

Reply:
It is necessary because the original anchor_generator has the same name as the config of anchor_generator, which is a config dict. So in the next iteration the config dict will be overwritten by the anchor_generator object and cause bugs.